CMake/resolve_dependency_modules/cpr.cmake

```diff
@@ -37,3 +37,5 @@ FetchContent_Declare(
 set(BUILD_SHARED_LIBS OFF)
 set(CPR_USE_SYSTEM_CURL OFF)
 FetchContent_MakeAvailable(cpr)
+# See issue #7964.
```

Comment:
Perhaps, add this bit of information to help readers understand this change:

"libcpr seems to have an issue in its CMakeLists.txt file when CPR_USE_SYSTEM_CURL=OFF.
It disables the BUILD_TESTING globally."

Reply:
I added a comment here.